{{cookiecutter.project_dirname}}/{{cookiecutter.django_settings_dirname}}/settings.py

```diff
@@ -142,6 +143,51 @@ class ProjectDefault(Configuration):
 
     # MEDIA_ROOT = BASE_DIR / "media"{% endif %}
 
+    {% if "s3" in cookiecutter.media_storage %}# Django Storages
+    # https://django-storages.readthedocs.io/en/latest/
+
+    AWS_S3_URL = values.Value()
+
+    @property
+    def AWS_S3_URL_PARTS(self):
+        """Return the AWS S3 URL parts."""
+        pattern = re.compile(
+            r"(?P<scheme>https?):\/\/(?P<access_key_id>.+?):(?P<secret_access_key>.+?)"
+            r"@(?P<host>.+?)(?::(?P<port>\d*?))?\/(?P<bucket_name>.+)"
+        )
+        return pattern.match(self.AWS_S3_URL).groupdict()
+
+    @property
+    def AWS_S3_ACCESS_KEY_ID(self):
```

Comment:
We might be able to simplify this back to what it used to be and get rid of the access key id / secret combination altogether if we made the bucket completely open.